main.tf

```diff
@@ -241,7 +240,6 @@ module "netapp" {
   prefix              = var.prefix
   resource_group_name = local.aks_rg.name
   location            = var.location
-  vnet_name           = module.vnet.name
```

Comment:
No longer needed? This is unclear.

Reply:
The value for `vnet_name` set here is not used in the `azurerm_netapp/main.tf`.  Removing this did not affect the functionality.